packages/modules/user/src/services/user-module.ts

```diff
@@ -311,26 +315,16 @@ export default class UserModuleService
     }
 
     const toCreate = data.map((invite) => {
+      const id = generateEntityId((invite as { id?: string }).id, "invite")
       return {
         ...invite,
-        expires_at: new Date(),
-        token: "placeholder",
+        id,
+        expires_at: new Date(Date.now() + this.config.expiresIn * 1000),
+        token: this.generateToken({ id, email: invite.email }),
       }
     })
 
-    const created = await this.inviteService_.create(toCreate, sharedContext)
-
-    const updates = created.map((invite) => {
-      return {
-        id: invite.id,
-        expires_at: new Date().setMilliseconds(
-          new Date().getMilliseconds() + this.config.expiresIn * 1000
-        ),
-        token: this.generateToken({ id: invite.id, email: invite.email }),
-      }
-    })
-
-    return await this.inviteService_.update(updates, sharedContext)
+    return await this.inviteService_.create(toCreate, sharedContext)
   }
```

Comment:
NOTE: This is a simplification, instead of doing a create and then an update, we only perform a create but the result remain unchanged